R/grab_mWater_malfunction_notes.R

```diff
@@ -0,0 +1,44 @@
+grab_mWater_malfunction_notes <- function(mWater_api_data){
+  
+  # Grab notes about sensor malfunction
+  malfunction_records <- mWater_api_data %>%
+    dplyr::filter(grepl("Sensor malfunction", visit_type, ignore.case = TRUE)) %>%
+    dplyr::select(start_DT, site, crew, which_sensor_malfunction, malfunction_end_dt, notes = visit_comments)
+  
+  #write to csv
+  # readr::write_csv(malfunction_records, "data/mWater_malfunction_records.csv")
+  
+  parameters <- c("Battery Level",
+                  "Baro",
+                  "Chl-a Fluorescence",
+                  "Depth",
+                  "DO",
+                  "External Voltage",
+                  "ORP",
+                  "pH",
+                  "Specific Conductivity",
+                  "Temperature",
+                  "Turbidity")
```

Comment:
This parameter list should be updated to remove some of these, no?
```suggestion
  parameters <- c("Chl-a Fluorescence",
                  "Depth",
                  "DO",
                  "ORP",
                  "pH",
                  "Specific Conductivity",
                  "Temperature",
                  "Turbidity")
```